CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 Enhancements
 
 - Upgraded Kotlin to `1.4.0`.
+- Improved the Bluetooth headset connection and audio change reliability by registering the `BluetoothHeadset.ACTION_CONNECTION_STATE_CHANGED` and `BluetoothHeadset.ACTION_AUDIO_STATE_CHANGED` intent actions.
```

Comment:
... intent actions instead of relying on android.bluetooth.BluetoothDevice intent actions.